generate-fit-profile/src/messages.rs

```diff
@@ -69,16 +71,15 @@ fn message_struct(message: &MessageDefinition) -> TokenStream {
         #[derive(Clone, Debug, PartialEq, PartialOrd, Serialize)]
         pub struct #ident {
             #( #struct_fields )*
+            /// All fields that are not defined in the profile.
+            pub unknown_fields: Vec<FitDataField>,
```

Comment:
Probably it would be better to use a map from field number to value here.  The field name will just be `unknown_field_x`.

Reply:
I agree, a map would simply access and they can just iterate over the values if they want them all.

Reply:
Done.